### DISCUSSION TITLE
[CGAL 6.1] How to use PMP::stitch_borders(surface, CGAL::parameters::maximum_distance(0.1)); ?

### DISCUSSION THREAD

**Discussion by ankofl** [CGAL/cgal#8614] [Q&A] [answered]
How do I call this method through the code?
![image](https://github.com/user-attachments/assets/c94943cf-be0b-44eb-926b-5cff8e62da65)
![image](https://github.com/user-attachments/assets/ce132af4-f5c2-4c7f-b21f-05f92c60c2c4)
I tried to do this with the same surface through the code:
```c
size_t stitched = PMP::stitch_borders(surface, CGAL::parameters::maximum_distance(0.1));
```
unlike CGAL_Lab, the edges did NOT collapse.

```
OFF
12 10 0

0 -1.2000000089999996 0
0 -1.2000000089999996 0
0.0099993899999901714 -1.2000000089999996 0
-0.12839449866666541 -2.0467452879644012 0
-1.1883939246666557 -2.0467452879644012 0
0.0099993899999901714 -2.6333333543333337 0
-0.12839449866666541 -1.2000000089999996 0
-0.12839449866666541 -2.0467452879644012 0
-1.4107966383333235 -1.2000000089999996 0
-1.1883939246666557 -1.2000000089999996 0
-1.4107966383333235 -2.6333333543333337 0
-1.1883939246666557 -2.0467452879644012 0
3  0 1 2
3  3 4 5
3  1 6 7
3  2 1 7
3  4 10 5
3  2 7 5
3  7 3 5
3  11 10 4
3  9 8 11
3  8 10 11
```

**Comment by ankofl** [comment]
upd:
I need the input mesh to sew all the slots and be unified and not split into separate components
![image](https://github.com/user-attachments/assets/9c33305f-9b3a-4f54-b377-d1fdfeb51c78)
![image](https://github.com/user-attachments/assets/3df77cf9-b9ea-42a1-9756-c52919a6ade4)

**Comment by sloriot** [comment, marked as answer]
You have to use something like this:
```
    std::vector< std::pair<halfedge_descriptor, halfedge_descriptor> > halfedges_to_stitch;
    PMP::collect_close_stitchable_boundary_edges(mesh, epsilon,mesh->points()), halfedges_to_stitch);
    PMP::stitch_borders(mesh, halfedges_to_stitch);
```

**Reply by ankofl** [reply]
It works great! Thank you very much!
